### ISSUE TITLE
Recent update to Windows Powershell breaks -Insecure flag in Connect-Safeguard cmdlet.

### ISSUE THREAD

**Issue by petrsnd** [OneIdentity/safeguard-ps#16]
Windows 10 and Windows Server 2016 are currently failing in `Connect-Safeguard` with the following error:

```Powershell
> Connect-Safeguard -Insecure 10.5.32.162 local Admin
Invoke-RestMethod : A system error occurred and has been logged.  Please try again later or contact your administrator.
At C:\Users\petrs\OneDrive\Documents\WindowsPowerShell\Modules\safeguard-ps\2.1.0.37\safeguard-ps.psm1:393 char:18
+ ...            (Invoke-RestMethod -Method GET -Uri "https://$Appliance/$( ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
```

**Comment by petrsnd** [comment]
The problem appears to be related to setting up a "Trust All" rule for server certificate verification.  This is a particularly finicky part of Powershell.  If you search online, you will see a ton of recommendations to just use:

```Powershell
> [System.Net.ServicePointManager]::ServerCertificateValidationCallback = {$true}
> Connect-Safeguard 10.5.32.162 local Admin
Invoke-RestMethod : The underlying connection was closed: An unexpected error occurred on a send.
At C:\Users\petrs\OneDrive\Documents\WindowsPowerShell\Modules\safeguard-ps\2.1.0.99999\safeguard-ps.psm1:393 char:18
+ ...            (Invoke-RestMethod -Method GET -Uri "https://$Appliance/$( ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
```

But, that also fails on more recent versions of Windows Powershell as you can see from the output above.

**Comment by petrsnd** [comment]
Posted write-up of this issue as a [StackOverflow question](https://stackoverflow.com/questions/46855241/ignoring-self-signed-certificates-from-powershell-invoke-restmethod-doesnt-work).

**Comment by petrsnd** [comment]
This appears to be a change in the rSTS code from which safeguard-ps has been fetching tokens.

**Comment by petrsnd** [comment]
The problem seems to have been a change in Safeguard's rSTS where the interface changed to the login controller we were using to discover supported identity providers.

**Comment by petrsnd** [comment]
Fixed by pull request #18
